ground/src/main/res/color/color_states_primary_button.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright 2023 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:color="@color/md_theme_on_surface_disabled" android:state_enabled="false"/>
```

Comment:
@JSunde Any idea how to get this color via the theme? It's not set in `styles.xml`, and not sure what ?attr it maps to.

Reply:
Looks styling is more complicated because we're using the wrong widget. Filed #1873 to track fast-follow!